client/src/components/App.jsx

```diff
@@ -1,15 +1,16 @@
-import Review from './components/Review.jsx';
-import Price from './components/Price.jsx';
-import Selection from './components/Selection.jsx';
-import Distribution from './components/Distribution.jsx';
-import Highlight from './components/Highlight.jsx';
+import React from 'react';
```

Comment:
I was thinking of trying to import {Component} instead of all of React.  I haven't gotten to try it yet but I wonder if it would allow more unused files to be pruned, or maybe we still need all of react, not just component.  Have you explored that at all?

Reply:
probably not a bad idea.